wayland-backend/src/rs/socket.rs

```diff
@@ -141,7 +151,7 @@ impl BufferedSocket {
             let written = self.socket.send_msg(bytes, fds)?;
             for &fd in fds {
                 // once the fds are sent, we can close them
-                let _ = ::nix::unistd::close(fd);
+                let _ = unsafe { libc::close(fd) };
```

Comment:
Couldn't [`rustix::io::close()`](https://docs.rs/rustix/0.38.17/rustix/io/fn.close.html) be used here instead?